.drone.jsonnet

```diff
@@ -505,7 +505,8 @@ local integration_bios_cgroupsv1 = Step('e2e-bios-cgroupsv1', target='e2e-qemu',
   SHORT_INTEGRATION_TEST: 'yes',
   WITH_UEFI: 'false',
   IMAGE_REGISTRY: local_registry,
-  WITH_CONFIG_PATCH: '[{"op": "add", "path": "/machine/install/extraKernelArgs/-", "value": "talos.unified_cgroup_hierarchy=0"}]',  // use cgroupsv1
+  // cgroupsv1 disabled, see: https://github.com/kubernetes/kubernetes/issues/120813
```

Comment:
The name of the step/test no longer makes sense now.

Reply:
Hopefully the upstream issue will get fixed, and we will re-enable

Reply:
i guess we can re-enable with newer pkgs